includes/frontend.mjs

```diff
@@ -132,6 +132,9 @@ export function addDownloadTSVListener(downloadTSV) {
         }
 
         const details = res.CATALOG.find(e => e.CodeName == element.CodeName);
+        if (typeof details == 'undefined') {
+          continue;
+        }
```

Comment:
Did you run into this?  I noticed I desynced the existing icon classifier and the catalog, such that this could occur.  I'm going to try to avoid that with the model for inferno though.

OK to have this here either way.  Just curious if you've found another bug I should note (i.e. a screenshot with an icon classified as an impossible item).

Reply:
i suspect this might have occured, because i had already switched to your new classifier and catalog.json in `fig.gs`, but had still been using the old, now symlinked `includes/catalog.json`.